Index: xournalpp/CMakeLists.txt
===================================================================
--- xournalpp.orig/CMakeLists.txt
+++ xournalpp/CMakeLists.txt
@@ -255,8 +255,6 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Linux"
 	install(FILES desktop/xournalpp.thumbnailer
 			DESTINATION ${CMAKE_INSTALL_PREFIX}/share/thumbnailers)
 
-	install(CODE "execute_process(COMMAND ${CMAKE_CURRENT_BINARY_DIR}/cmake/postinst configure)")
-
 endif ()
 
 # Uninstall target
