--- a/dwarfdump/Makefile.in	2015-09-30 18:03:08.690113457 +0200
+++ b/dwarfdump/Makefile.in	2015-09-30 18:02:51.866886308 +0200
@@ -118,10 +118,10 @@
 
 # Use static libdwarf.a
 tag_tree_build:  $(srcdir)/tag_tree.c $(DIRINC)/dwarf.h $(HEADERS) tag_common.o makename.o common.o trivial_naming.o dwgetopt.o
-	$(CC) $(CFLAGS) $(srcdir)/tag_tree.c tag_common.o common.o makename.o trivial_naming.o dwgetopt.o ../libdwarf/libdwarf.a $(LDFLAGS) -o tag_tree_build 
+	$(CC) $(CFLAGS) $(srcdir)/tag_tree.c tag_common.o common.o makename.o trivial_naming.o dwgetopt.o ../libdwarf/libdwarf.a $(LDFLAGS) -Wl,-rpath ../libdwarf -o tag_tree_build 
 
 tag_attr_build:  $(srcdir)/tag_attr.c $(DIRINC)/dwarf.h $(HEADERS) tag_common.o makename.o common.o trivial_naming.o dwgetopt.o
-	$(CC) $(CFLAGS) $(srcdir)/tag_attr.c tag_common.o common.o makename.o trivial_naming.o dwgetopt.o ../libdwarf/libdwarf.a $(LDFLAGS) -o tag_attr_build 
+	$(CC) $(CFLAGS) $(srcdir)/tag_attr.c tag_common.o common.o makename.o trivial_naming.o dwgetopt.o ../libdwarf/libdwarf.a $(LDFLAGS) -Wl,-rpath ../libdwarf -o tag_attr_build 
 
 tmp-tt-table.c: $(srcdir)/tag_tree.list tag_tree_build
 	# gcc -E tag_tree.list does not work, so use a .c name
