diff --git a/configure.ac b/configure.ac
index 4b1241b1..31259f7f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -44,7 +44,7 @@ if test "x$remember_set_CFLAGS" = "x"; then
 	fi
 fi
 
-CFLAGS="$CFLAGS $OPENMP_CFLAGS"
+CFLAGS="-Werror=unknown-pragmas -Werror=unused-label -fno-common $CFLAGS $OPENMP_CFLAGS"
 CXXFLAGS="$CFLAGS"
 
 dnl Set PACKAGE_SOURCE_DIR in config.h.
diff --git a/src/algos/Def_Wavelet.h b/src/algos/Def_Wavelet.h
index 3dd48f80..88954aec 100644
--- a/src/algos/Def_Wavelet.h
+++ b/src/algos/Def_Wavelet.h
@@ -58,24 +58,24 @@ typedef struct 	{
 	int Size;
 	float Freq_Coup; /* Frequency cutt-off */
 	float *Data;
-	        } pyramid_cf_des;
+} pyramid_cf_des;
 
 /* Data  structure for an algorithm without reduction of sampling */
 typedef struct 	{
 	float *Data;
 	float Freq_Coup;  /* Frequency cutt-off */
-	        } pave_f_des;
+} pave_f_des;
 
 
 /* Data structure for Mallat's algorithm */
 struct mallat_plan_des	{
 	int Nl,Nc;
-        float *Coef_Horiz;
-        float *Coef_Diag;
-        float *Coef_Vert;
-        float *Low_Resol;
-        struct mallat_plan_des *Smooth_Imag;
-        } mallat_plan_des;
+	float *Coef_Horiz;
+	float *Coef_Diag;
+	float *Coef_Vert;
+	float *Low_Resol;
+	struct mallat_plan_des *Smooth_Imag;
+};
 
 /* Data structure for a wavelet transform */
 typedef struct 	{
@@ -91,29 +91,29 @@ typedef struct 	{
 	pyramid_f_des Pyramid;
 	pave_f_des Pave;
 	struct mallat_plan_des Mallat;
-	         } wave_transf_des;
+} wave_transf_des;
 
 /* Data structure for image information */
 typedef struct {
-        float Sigma;
-        float Mean;
-        float Min, Max;
-        float Energ, Entrop;
-        float Correl_Plan[MAX_PLAN_WAVELET];
-	         } plan_info_des;
+	float Sigma;
+	float Mean;
+	float Min, Max;
+	float Energ, Entrop;
+	float Correl_Plan[MAX_PLAN_WAVELET];
+} plan_info_des;
 
 /* Filtering */
 
-    /* Thresholding */
+/* Thresholding */
 #define FILTER_TRESHOLD 1     
 
-    /* Adaptative thresholding */
+/* Adaptative thresholding */
 #define FILTER_HIERARCHICAL_TRESHOLD 2
 
-    /* Hierarchical Wiener filtering */
+/* Hierarchical Wiener filtering */
 #define FILTER_HIERARCHICAL 3
 
-    /* Multiresolution Wiener filtering */
+/* Multiresolution Wiener filtering */
 #define FILTER_MULTI_RES_WIENER 4
 
 #define TO1_FRENCH 1
diff --git a/src/core/command.c b/src/core/command.c
index f06bac73..476fef23 100644
--- a/src/core/command.c
+++ b/src/core/command.c
@@ -213,6 +213,7 @@ static command commands[] = {
 	{"",0,"",0, STR_NONE, FALSE}
 };
 
+
 int process_load(int nb){
 	char filename[256];
 	int retval, i;
diff --git a/src/core/siril.h b/src/core/siril.h
index 6a56f9b1..817e5831 100644
--- a/src/core/siril.h
+++ b/src/core/siril.h
@@ -275,7 +275,7 @@ struct layer_info_struct {
 	display_mode rendering_mode;	// defaults to NORMAL_DISPLAY
 };
 
-typedef enum { SEQ_REGULAR, SEQ_SER,
+typedef enum { SEQ_REGULAR, SEQ_SER, /*SEQ_CUBE,*/
 #ifdef HAVE_FFMS2
 	SEQ_AVI,
 #endif
diff --git a/src/gui/save_dialog.h b/src/gui/save_dialog.h
index 9a8f35d1..471904bb 100644
--- a/src/gui/save_dialog.h
+++ b/src/gui/save_dialog.h
@@ -14,6 +14,6 @@ struct savedial_data {
 
 enum {
 	PAGE_TIFF, PAGE_JPG, PAGE_FITS, PAGE_MISC
-} confirm;
+};
 
 #endif /* SRC_GUI_SAVE_DIALOG_H_ */
